python/ray/_raylet.pyx

```diff
@@ -729,8 +729,9 @@ cdef execute_task(
             exit.is_ray_terminate = True
             raise exit
 
-# return a protobuf-serialized ray_exception
+# return a LocalMemoryBuffer containing a protobuf-serialized RayException
```

Comment:
Just leave it alone? I think the more comment, the better haha. So as your other comments about the comment.

Reply:
I don't think more comments == better. Comment can be easily diverged from implementation because people don't usually fix it when they change code, so we should avoid having redundant comments (if we have a chance).

Reply:
Same for the below one. If somebody changes the return type of `to_bytes`, it will just add cognitive overhead (since the comment and code are now different), which is a tech debt we should avoid as much as possible.

Reply:
OK, I agree with you. I'll remove them.